src/Altinn.Profile.Integrations/Extensions/WebApplicationExtensions.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System.Diagnostics.CodeAnalysis;
+
+using Altinn.Profile.Integrations.Persistence;
+
+using Microsoft.AspNetCore.Builder;
+using Microsoft.Extensions.Configuration;
+
+using Yuniql.AspNetCore;
+using Yuniql.PostgreSql;
+
+namespace Altinn.Profile.Integrations.Extensions;
+
+/// <summary>
+/// Extension class for web application
+/// </summary>
+[ExcludeFromCodeCoverage]
+public static class WebApplicationExtensions
+{
+    /// <summary>
+    /// Configure and set up db
+    /// </summary>
+    /// <param name="app">app</param>
+    /// <param name="isDevelopment">is environment dev</param>
+    /// <param name="config">the configuration collection</param>
+    public static void SetUpPostgreSql(this IApplicationBuilder app, bool isDevelopment, IConfiguration config)
+    {
+        PostgreSqlSettings? settings = config.GetSection("PostgreSQLSettings")
+            .Get<PostgreSqlSettings>()
+            ?? throw new ArgumentNullException(nameof(config), "Required PostgreSQLSettings is missing from application configuration");
+
+        if (settings.EnableDBConnection)
+        {
+            ConsoleTraceService traceService = new() { IsDebugEnabled = true };
```

Comment:
So this line should run in any environment, and not just in local dev? Just making sure that the "debug enabled" won't cause troubles in prod

Reply:
Debug is disabled by default.